wazimap_ng/profile/serializers/category_serializer.py

```diff
@@ -1,13 +1,20 @@
 from rest_framework import serializers
 
 from .. import models
+from wazimap_ng.datasets.models.dataset import Dataset
```

Comment:
@emre2038 We can remove this import if we are using it

Reply:
done